Druid/DRCompilationBuildUnit.class.st

```diff
@@ -0,0 +1,39 @@
+Class {
+	#name : #DRCompilationBuildUnit,
```

Comment:
Maybe should this be called compilation unit builder?

Reply:
Done